.editorconfig

```diff
@@ -0,0 +1,10 @@
+# EditorConfig is awesome: https://EditorConfig.org
+
+root = true
+
+[*]
+charset = utf-8
+end_of_line = lf
+indent_style = tab
+insert_final_newline = true
+trim_trailing_whitespace = true
```

Comment:
I think this doesn't affect the code, only the visualization. That's why I left it out as a personal preference.

`tab_width` doesn't need to be set since it already defaults to the same value as indent_size.

Reply:
This is done too. Since now indent uses spaces, indent_size is important.